integration-tests/pom.xml

```diff
@@ -29,6 +29,7 @@
         <module>kafka</module>
         <module>jpa</module>
         <module>jpa-postgresql</module>
+        <module>jpa-postgresql-vault</module>
```

Comment:
As said, I would simplify things and make the module called `vault`.

Reply:
done